ginga/examples/configs/channel_Image.cfg

```diff
@@ -0,0 +1,118 @@
+#
+# Image channel preferences file
+#
+# Place this in file under ~/.ginga with the name "channel_Image.cfg" for
+# channel name "Image". This file WILL BE OVERWRITTEN by the Preferences
+# plugin (see below). NOT ALL PARAMETERS IN GUI CAN BE ADJUSTED HERE.
+#
+# Alternately, the settings can be adjusted via Preferences local plugin
+# within Ginga. NOT ALL PARAMETERS CAN BE ADJUSTED VIA GUI but users can
+# normally ignore the hidden ones.
+#
+
+# ---------------
+# Color Distribution
+
+color_algorithm = 'linear'
+
+# ---------------
+# Color Mapping
+
+color_map = 'gray'
+color_hashsize = 65535
+intensity_map = 'ramp'
+
+# ---------------
+# Auto Cuts
+
+autocut_method = 'zscale'
+autocut_params = {}
+cuts = (0.0, 0.0)
+
+# ---------------
+# Transform
+
+flip_x = False
+flip_y = False
+swap_xy = False
+rot_deg = 0.0
+
+# ---------------
+# WCS
+
+wcs_coords = 'icrs'
+wcs_display = 'sexagesimal'
+
+# ---------------
+# Zoom
+
+zoom_algorithm = 'step'
+zoom_rate = 1.4142135623730951
+zoomlevel = 1.0
+scale_x_base = 1.0
+scale_y_base = 1.0
+scale = (1.0, 1.0)
+scale_min = 1e-05
+scale_max = 10000.0
+interpolation = 'basic'
+
+# ---------------
+# Panning
+
+pan = (1.0, 1.0)
+pan_coord = 'data'
```

Comment:
I think the "WCS sexagesimal" box just controls how the pan wcs is formatted in the `Preferences` plugin.  It's not actually a channel setting.  It could be made into a `plugin_Preferences.cfg` preference though.